pom.xml

```diff
@@ -534,7 +585,7 @@
                     <testSource>17</testSource>
                     <testTarget>17</testTarget>
                 </configuration>
-            </plugin>
+            </plugin>-->
```

Comment:
We can remove this commented-out section, I forgot to remove it in the original PR...